models/spike_generator.h

```diff
@@ -249,18 +250,22 @@ class spike_generator : public DeviceNode
     bool precise_times_;
 
     //! Allow and round up spikes not on steps; irrelevant if precise_times_
+    bool allow_offgrid_times_;
+    //! Same as allow_offgrid_times_, here for backward compatibility
     bool allow_offgrid_spikes_;
 
     //! Shift spike times at present to next step
     bool shift_now_spikes_;
 
+    bool deprecation_warning_issued_;
```

Comment:
We may have deprecation warnings also on other parameters in other models in the future. We should find a general way of handling them, so we don't need to implement it in each node class that has a deprecated parameter. Maybe a general class for handling deprecations (essentially a map from names to flags), and then a member of this class in the `Node` base class.